devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java

```diff
@@ -335,7 +335,9 @@ private void addSelfWithLocalDeps(Project project, GradleDevModeLauncher.Builder
         if (!visited.add(project.getPath())) {
             return;
         }
-        final Configuration compileCp = project.getConfigurations().findByName(JavaPlugin.COMPILE_CLASSPATH_CONFIGURATION_NAME);
+        final Configuration compileCp = project.getConfigurations().create("compileCp")
```

Comment:
To be consistent we could use QuarkusPlugin.DEV_MODE_CONFIGURATION_NAME, right?

Reply:
It looks like it should already be available?

Reply:
Yes and it will also include ˋquarkusDev` dependency. 
This configuration only includes runtime classpath plus ˋcompileOnly` artifacts. I will add `runtimeOnly` artifacts too.